### PR TITLE
Release 4.0.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by root-expert** [comment]
Μaybe we can generate changelog for the first time?

**Comment by bastelfreak** [comment]
ah, I generated it but forgot to do a git add

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+**Closed issues:**
+
+- Debian's rclocal service name is rc-local.service not rc.local.service [\#27](https://github.com/voxpupuli/puppet-rclocal/issues/27)
+
+**Merged pull requests:**
+
+- cleanup .fixtures.yml [\#35](https://github.com/voxpupuli/puppet-rclocal/pull/35) ([bastelfreak](https://github.com/bastelfreak))
+- puppetlabs/stdlib: Allow 8.x [\#32](https://github.com/voxpupuli/puppet-rclocal/pull/32) ([bastelfreak](https://github.com/bastelfreak))
+- modulesync 5.1.0 [\#30](https://github.com/voxpupuli/puppet-rclocal/pull/30) ([bastelfreak](https://github.com/bastelfreak))
```
Comment (root-expert) [inline CHANGELOG.md:30]: I added skip-changelog

Comment (bastelfreak) [reply CHANGELOG.md]: updated it

**Review by root-expert — APPROVED** [review]

**Pull request merged**